.github/CODEOWNERS

```diff
@@ -60,6 +55,6 @@
 /tests/modules/images/dominant-color                                        @pbearne @spacedmonkey
 /tests/testdata/modules/images/dominant-color                               @pbearne @spacedmonkey
 
-# Module: Full Page Cache Health Check
-/modules/site-health/audit-full-page-cache                                  @manuelRod @westonruter
-/tests/modules/site-health/audit-full-page-cache                            @manuelRod @westonruter
+# Module: Audit Full Page Cache
+/modules/object-cache/audit-full-page-cache                                  @manuelRod @westonruter
+/tests/modules/object-cache/audit-full-page-cache                            @manuelRod @westonruter
```

Comment:
This isn't really an `object-cache` module.

Reply:
@tillkruss I'd still say this is the closest focus area it fits into. Potentially we could formally broaden the focus to "Caching", if you're open to it. But even if not, I think for now this module fits best here, and to be fair, it will be removed soon anyway as it was already merged into core. :)

Reply:
👍

Reply:
I don't think "Caching" makes sense. Let's go with this then.